recipes/libedit/0001-Also-look-in-libtinfo-for-tgetent.patch

```diff
@@ -0,0 +1,100 @@
+diff -urN work.old/configure work/configure
+--- work.old/configure	2017-08-22 14:16:00.621910386 +0100
++++ work/configure	2017-08-22 14:17:20.482361609 +0100
+@@ -586,7 +586,7 @@
+ 
+ # Identity of this package.
+ PACKAGE_NAME='libedit'
+-PACKAGE_TARNAME='libedit-20170329'
++PACKAGE_TARNAME='libedit-20170822'
```

Comment:
Why this change?

Reply:
I think this is just to fool the package.  I picked up this patch from another anaconda libedit package.